store/tikv/gc_worker.go

```diff
@@ -152,6 +129,44 @@ func (w *GCWorker) start(ctx goctx.Context) {
 	}
 }
 
+func (w *GCWorker) createSession() {
+	for {
+		time.Sleep(time.Second)
+
+		if !w.storeIsBootstrapped() {
+			log.Warnf("[gc worker] wait for store bootstrapping")
+			continue
+		}
+		var err error
+		w.session, err = tidb.CreateSession(w.store)
+		if err != nil {
+			log.Warnf("[gc worker] create session err: %v", err)
+			continue
+		}
+		// Disable privilege check for gc worker session.
+		privilege.BindPrivilegeManager(w.session, nil)
+		return
+	}
+}
+
+func (w *GCWorker) tick(ctx goctx.Context) {
+	isLeader, err := w.checkLeader()
+	if err != nil {
+		log.Warnf("[gc worker] check leader err: %v", err)
+		return
```

Comment:
Need to continue the loop event if we encounter an error, so just print it then go on.